### DISCUSSION TITLE
reviewing-every-jingle-jam-2023-game/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#11] [Comments]
# reviewing-every-jingle-jam-2023-game/

Every December since 2011, The Yogscast have organised the “Jingle Jam”, a collection of streams with an associated bundle with £1100+ of great games to raise money for charity. I’m reviewing all 75+ of them, hopefully finding a few new favourites! Here’s part 1 with the first 30 games.

https://jakelee.co.uk/reviewing-every-jingle-jam-2023-game/

**Comment by deathybrs** [comment]
By the way, this list is not "ALL," it is missing some.

**Reply by JakeSteam** [reply]
Yep, hence "Part 1" being in the post's title and description! [Part 2 is here](https://jakelee.co.uk/reviewing-every-jingle-jam-2023-game-2/), the final part (games 61-75) should be out soon :)

**Reply by deathybrs** [reply]
AH.  I did miss that part, thanks!
